_layouts/base.html.haml

```diff
@@ -24,6 +24,7 @@ change_frequency: monthly
     %link(rel="stylesheet"){:href => relative("/website/highlight/highlight.css")}
     %link(rel="stylesheet"){:href => relative("/website/tabzilla/tabzilla.css")}
     %link(rel="stylesheet"){:href => relative("/website/optaplannerWebsite.css")}
+    %link(rel="stylesheet"){:href => "https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css"}
```

Comment:
ok, thanks for warning. Let me know when it's good to merge again (even though not all issues will be solved, I'd like to go to a stable state today, so the rest of the improvements can be delayed to when there's more time)

As for the mutliline h3 headers: that's fine, as I think I can go through them pretty fast to split them into a header and the first paragraph afterwards.